packages/esm-patient-chart-app/src/ui-components/patient-chart.component.scss

```diff
@@ -56,5 +57,14 @@ $actionPanelExpandedOffset: $actionNavOffset + $actionPanelOffset;
   display: flex;
   width: 100%;
   padding-right: $actionNavOffset;
-  flex-direction: column;
+  flex-direction: row;
+}
+
+.inActiveDesktopWorkspace{
```

Comment:
This is rather inflexible CSS design. It's usually not how its done. You also don't include this with a notion of "desktop", but rather, for instance "activeInlineWorkspace".

Also usually this is where CSS media queries shine.